src/app/index.ts

```diff
@@ -331,17 +302,12 @@ module.exports = yo.extend({
     this.log(`      ${chalk.green('Congratulations!')} Your add-in has been created! Your next steps:\n`);
     this.log(`      1. Go the directory where your project was created:\n`);
     this.log(`         ${chalk.bold('cd ' + this._destinationRoot)}\n`);
-    this.log(`      2. Trust the Self-Signed Certificate for your local web server (if you haven't already done that).`);
-    this.log(`         For more information, visit https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md.\n`);
-    this.log(`      3. Start the local web server:\n`);
+    this.log(`      2. Start the local web server and sideload the add-in:\n`);
```

Comment:
It would be better to replace step 3 with step 2 (open in VS Code), and then have the next step be to use `Terminal, Run Task, "Dev Server"` to run the local web server in VS Code. Instead of sideload, "Start without Debugging" can be used to launch the host and load the add-in.

Reply:
Note that for Excel Custom Functions, "Start Debugging" should be used instead of "Start Without Debugging" because you can debug the custom functions runtime with VS Code. 

@kbrandl Perhaps we should provide a link to docs that explain how to do these steps. I think there is value in also providing some steps here as some users may not click the link.

Reply:
We can also merge these changes, and then continue to refince this afterwards...